### ISSUE TITLE
Add option to enable window staying on top 

### ISSUE THREAD

**Issue by ishboo** [fragment-nine/Show-Saver#20]

**Comment by ishboo** [comment]
Is this what the "Hold Window" button is supposed to do? It's not working for me

**Comment by andrewgange** [comment]
Yes - the hold window button toggles the "Always on Top" toggle from the perform window comp. is it not toggling on your end? Working on my end

**Comment by andrewgange** [comment]
looks like for it to be toggled you have to leave the UI, then reenter it and the button corresponds. I had the button toggled on and it was holding, then I toggled it off and it was still holding. Unsure the fix to this ATM besides hitting ESC and then F1 after trigger

**Comment by ishboo** [comment]
The selection works but it seems to require closing the perform window and relaunching it
